bakerydemo/templates/base/home_page.html

```diff
@@ -4,121 +4,99 @@
+                    {% else %}
                     {{ page.hero_cta }}
-                {% endif %}
+                    {% endif %}
+                </div>
             </div>
         </div>
     </div>
-</div>
 
-<div class="container">
-    <div class="row promo-row">
-        <div class="col-sm-5 promo">
-            {% if page.promo_image or page.promo_title or page.promo_text %}
-                <figure>{% image page.promo_image fill-200x200-c100 %}</figure>
-                <div class="promo-text">
-                    <h3>{{ page.promo_title }}</h3>
-                    {{ page.promo_text|safe }}
-                </div>
-            {% endif %}
-        </div>
-
-        <div class="col-sm-6 col-sm-offset-1 feature-1">
-            {% if page.featured_section_1 %}
-            <h2>{{ page.featured_section_1_title }}</h2>
-                <div class="featured-children">
-                    {% for childpage in page.featured_section_1.specific.children|slice:"4" %}
-                        <li>
-                            <div class="row">
-                                <div class="col-xs-4">
-                                    <a href="{{childpage.url}}">
-                                        <figure>
-                                            {% image childpage.image fill-180x140-c100 %}
-                                        </figure>
-                                    </a>
-                                </div>
-                                <div class="col-xs-8">
-                                    <h3><a href="{{childpage.url}}">{{childpage.title}}</a></h3>
-                                </div>
-                            </div>
-                        </li>
+    <div class="container">
+        <div class="row promo-row">
+            <div class="featured-cards col-sm-5 col-sm-offset-1">
+                {% if page.featured_section_1 %}
+                <h2 class="featured-cards__title">{{ page.featured_section_1_title }}</h2>
+                <ul class="featured-cards__list">
+                    {% for childpage in page.featured_section_1.specific.children|slice:"3" %}
+                    {% include "includes/card/listing-card.html" with page=childpage %}
                     {% endfor %}
+                </ul>
+                <a class="featured-cards__link" href="/breads">
+                    <span>View more of our breads</span>
+                    {% include "includes/chevron-icon.html" with class="featured-cards__chevron-icon" %}
+                </a>
+                {% endif %}
+            </div>
+
+            <div class="col-sm-6 promo">
+                {% if page.promo_image or page.promo_title or page.promo_text %}
+                <div class="col-lg-10 promo-text">
+                    {% if page.promo_title %}
+                    <h2>{{ page.promo_title }}</h2>
+                    {% endif %}
+                    {% if page.promo_text %}
+                    {{ page.promo_text|safe }}
```

Comment:
Could we try removing the safe?

Reply:
Sadly, this causes the following glitch: 

<img width="544" alt="image" src="https://user-images.githubusercontent.com/18164832/169990928-e09222b9-d591-4c52-9d1d-45e8e3da661b.png">

Reply:
Yes, this means we should be using `|richtext` rather than `|safe`.